clippy_lints/src/literal_representation.rs

```diff
@@ -538,7 +557,7 @@ impl LiteralRepresentation {
             // Lint for Literals with a hex-representation of 2 or 3 digits
             let f = &digits[0..1]; // first digit
             let s = &digits[1..]; // suffix
-            // Powers of 2
+                                  // Powers of 2
```

Comment:
This indentation on this comment looks wrong. The comment refers to the lines below it, right?

Reply:
The misplacement of the comment was caused by rustfmt misinterpreting the comment's relationship the previous comments.  I added a space to fix the spacing.